docs/guides/sdk/getting-started.md

```diff
@@ -0,0 +1,226 @@
+
+```ts
+import { useDirectus, readItem } from '@directus/sdk@beta';
+import { rest } from '@directus/sdk@beta/composable';
+
+type Schema = {
+	articles: { title: string; content: string };
+};
+
+const client = useDirectus<Schema>('http://directus.example.com').use(rest());
+const result = await client.request(readItem('articles', 5));
+```
+
+#### Read all items
+
+```js
+import { useDirectus, readItems } from '@directus/sdk@beta';
+import { rest } from '@directus/sdk@beta/composable';
+
+const client = useDirectus('http://directus.example.com').use(rest());
+const result = await client.request(readItems('articles'));
+```
+
+#### Read specific fields
+
+```js
+import { useDirectus, readItems } from '@directus/sdk@beta';
+import { rest } from '@directus/sdk@beta/composable';
+
+const client = useDirectus('http://directus.example.com').use(rest());
+const result = await client.request(
+	readItems('articles', {
+		fields: ['id', 'title'],
+	})
+);
+```
+
+#### Read all fields
+
+```js
+import { useDirectus, readItems } from '@directus/sdk@beta';
+import { rest } from '@directus/sdk@beta/composable';
+
+const client = useDirectus('http://directus.example.com').use(rest());
+const result = await client.request(
+	readItems('articles', {
+		fields: ['*'],
+	})
+);
+```
+
+#### Read nested fields
+
+```js
+import { useDirectus, readItems } from '@directus/sdk@beta';
+import { rest } from '@directus/sdk@beta/composable';
+
+const client = useDirectus('http://directus.example.com').use(rest());
+const result = await client.request(
+	readItems('articles', {
+		fields: ['*', { author: ['*'] }],
+	})
+);
+```
+
+### Using GraphQL
+
+Add the `graphql()` composable to the client, this enables the `.query(...)` method to query the collection.
+
+For example, to make a request to an `articles` collection.
+
+```ts
+import { useDirectus } from '@directus/sdk@beta';
+import { graphql } from '@directus/sdk@beta/composable';
+
+type Article = {
+	title: string;
+	content: string;
+};
+
+type Schema = {
+	articles: Article;
+};
+
+const client = useDirectus<Schema>('http://directus.example.com').use(graphql());
+const result = await client.query<Article[]>(`
+    query {
+        articles {
+            id
+            title
+            content
+        }
+    }
+`);
+```
+
+> Feel free to customize the queries according to your project's GraphQL schema.
+
+## Next Steps
+
+You have successfully created a composable client using the Directus SDK.
+
+As you progress, you may want to explore other available features like real-time updates, and GraphQL subscriptions.
+
+Keep an eye on our documentation as we continue to enhance and expand the SDK.
+
+If you encounter any issues or have questions, don't hesitate to reach out to us in our
+[Discord community](https://directus.chat/).
+
+## Backward Compatibility
+
+The older Directus SDK will still work and your existing applications that use the older version will continue to
+function without any issues.
```

Comment:
This feels wrong. I'm reading it as "this is not backwards compatible" (because it isn't) "but feel free to keep using the older versions".

Reply:
I changed the heading to "Older SDK Version". I think that makes the content more understandable